test/test_command.py

```diff
@@ -161,15 +161,6 @@ class ServerConfig:
 server_config = pytest.mark.parametrize("qtile", [ServerConfig], indirect=True)
 
 
-@server_config
-def test_item_types(qtile):
```

Comment:
Can we change this to use the new normalize_item() instead of just dropping the test?